package.json

```diff
@@ -1,42 +1,76 @@
 {
-  "name": "node-red-contrib-kolibri",
-  "version": "0.2.2",
-  "description": "Node-Red nodes for connecting to a Kolibri Broker",
-  "author": "HMS Networks",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/hms-networks/node-red-contrib-kolibri"
+  "_from": "node-red-contrib-kolibri@0.2.2",
```

Comment:
from _id to _where should be removed. Some local configuration there.

Reply:
Done